src/workerd/api/worker-rpc.c++

```diff
@@ -501,17 +501,18 @@ JsRpcPromiseAndPipleine callImpl(jsg::Lock& js,
       kj::Maybe<StreamSinkFulfiller> paramsStreamSinkFulfiller;
 
       KJ_IF_SOME(args, maybeArgs) {
-        // This is a function call with arguments.
-        kj::Vector<jsg::JsValue> argv(args.Length());
-        for (int n = 0; n < args.Length(); n++) {
-          argv.add(jsg::JsValue(args[n]));
-        }
-
         // If we have arguments, serialize them.
         // Note that we may fail to serialize some element, in which case this will throw back to
         // JS.
-        if (argv.size() > 0) {
-          serializeJsValue(js, js.arr(argv.asPtr()), [&](capnp::MessageSize hint) {
+        if (args.Length() > 0) {
+          // This is a function call with arguments.
+          v8::LocalVector<v8::Value> argv(js.v8Isolate, args.Length());
+          for (int n = 0; n < args.Length(); n++) {
+            argv[n] = args[n];
+          }
+          auto arr = v8::Array::New(js.v8Isolate, argv.data(), argv.size());
```

Comment:
Can we just eliminate LocalVector in here, and just use v8::array constructor with length, and update it? Doesn't this make an additional copy?

Reply:
No, using individual calls to `->Set(...)` would be a lot slower. The `v8::LocalVector<T>` is not copying anything. This form is still more performant